crates/chia-consensus/src/spendbundle_conditions.rs

```diff
@@ -93,6 +93,9 @@ mod tests {
     use rstest::rstest;
     use std::fs::read;
 
+    const QUOTE_EXECUTION_COST: u64 = 20;
+    const QUOTE_BYTE_COST: u64 = 2 * TEST_CONSTANTS.cost_per_byte;
```

Comment:
May  as well use the constant?

```suggestion
    const QUOTE_BYTE_COST: u64 = QUOTE_BYTES * TEST_CONSTANTS.cost_per_byte;
```